zipline/finance/performance/tracker.py

```diff
@@ -77,7 +78,7 @@ class PerformanceTracker(object):
     """
     Tracks the performance of the algorithm.
     """
-    def __init__(self, sim_params, trading_calendar, env):
```

Comment:
Do we still need separate trading calendar and env parameters if this class is accepting a data portal now?